docs/tutorials/index.rst

```diff
@@ -11,3 +11,13 @@ the IDAES tutorials `Short introduction to Python and Pyomo <https://idaes.githu
 
 The tutorials included with WaterTAP are available in the "tutorials" directory
 on the WaterTAP GitHub Homepage: `<https://github.com/watertap-org/watertap/tree/main/tutorials>`_.
+
+.. note::
+    For important details on running tutorial notebooks, refer to `this guide<https://watertap.readthedocs.io/en/stable/how_to_guides/notebooks.html#online-using-binder-org>`_.
```

Comment:
```suggestion
    For important details on running tutorial notebooks, refer to :ref:`this guide<notebooks-online>`.
```

Reply:
@adam-a-a I think the internal reference would be a better way to link to the guide, as it will ensure that the link's source remains in sync with its target. However, note that this requires the reference in `docs/how_to_guides/notebooks.rst` to be changed from `.. _noteboks-online:` to `.. _notebooks-online:` (I misspelled when I added that section).